src/components/notifications/dto/notification.dto.ts

```diff
@@ -0,0 +1,35 @@
+import { Field, InterfaceType, ObjectType } from '@nestjs/graphql';
+import { keys as keysOf } from 'ts-transformer-keys';
+import { Resource, SecuredProps } from '~/common';
+import { LinkTo, RegisterResource } from '~/core/resources';
+
+@RegisterResource()
+@InterfaceType({
+  implements: [Resource],
+})
+export class Notification extends Resource {
+  static readonly Props = keysOf<Notification>();
+  static readonly SecuredProps = keysOf<SecuredProps<Notification>>();
+
+  readonly owner: LinkTo<'User'>;
+
+  @Field(() => Boolean)
+  readonly unread: boolean;
```

Comment:
Or would the flipped `read` be better?

Reply:
That's a tough one; I was thinking about that before I saw your comment.  Typically I would say to flip it; positive approach over double-negative and all that...but this case is so well known and little thought of as "Oh, these are my 'read' notifications."  Of course, that's from a UI/visual perspective, so maybe that shouldn't play in to the model.  Kinda torn here...just typing it out so far.

Reply:
Put me down as another one who's torn the name, but I would keep it as is.

Reply:
I'm fine with keeping it as `unread` as well.